CHANGELOG.md

```diff
@@ -117,7 +117,7 @@ recommended to use the method from `Deployer` rather than the function dirrectly
 - Allow token to be empty.
 - Throw error when issue inner transactions in clear program. Fixed in [#667](https://github.com/scale-it/algo-builder/pull/667).
 - Parameters in `extract*` opcodes can greater than uint8. Fixed in [#666](https://github.com/scale-it/algo-builder/pull/666).
-
+- Cannot send duplicate transaction in group transaction. 
```

Comment:
```suggestion
- Restirct duplicate transaction in group transaction. 
```

Reply:
This case is force. If user send duplicate transaction it's will throw error. I worry use `Restrict` will make user confuse.

Reply:
This is for our own track of changes.